config/install/field.field.media.google_form.field_media_google_form.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.media.field_media_google_form
+    - media.type.google_form
+id: media.google_form.field_media_google_form
+field_name: field_media_google_form
+entity_type: media
+bundle: google_form
+label: 'Google Form'
```

Comment:
IMO, this should be 'Google Form URL'.  The meaning could be unclear in the UI.

Reply:
That is a good question. We should bring that up with Ishita.